README.md

```diff
@@ -54,42 +82,43 @@ import magnolify.guava.semiauto._
 FunnelDerivation[MyCaseClass]
 ```
 
-Case class type conversion must be done explicitly.
+Case class type conversion must be called explicitly, specifying the case class used for conversion, rather than deriving entirely with implicits.
 
 ```scala
+// an annotated example with Avro GenericRecord
 import magnolify.avro._
-val t = AvroType[MyCaseClass]
-val r = t.to(MyCaseClass(...))
-val c = t.from(t)
+import org.apache.avro.generic.GenericRecord
+val typeclass = AvroType[MyCaseClass] // AvroType typeclass instance defines to and from conversions between Case Class and GenericRecord
+val genericRecord: GenericRecord = typeclass.to(MyCaseClass(...)) // instantiate a case class, then pass in
+val caseClass: MyCaseClass = typeclass.from(genericRecord) // roundtrip
 
-t.schema // Avro Schema
+typeclass.schema // Avro Schema
 implicit val uriField = AvroField.from[String](URI.create)(_.toString) // custom field type
-```
 
-```scala
+// The other three converters work similarly to Avro: 
+
+// an example with BigQuery TableRow
 import magnolify.bigquery._
-val t = TableRowType[MyCaseClass]
-val r = t.to(MyCaseClass(...))
-val c = t.from(t)
+val typeclass = TableRowType[MyCaseClass] 
```

Comment:
It's using `t` in some places & `typeclass` here. Make them consistent? Doesn't matter which way.

I was being lazy and using `t`, `r`, `c` for type class, record (Avro, BQ, etc.) & case class.

Reply:
I started off trying to convert the names to all longer/more meaningful, but missed some towards the end. I will fix to use longer names throughout